tests/TestCase/Network/Http/Adapter/StreamTest.php

```diff
@@ -13,8 +13,8 @@
  */
 namespace Cake\Test\TestCase\Network\Http\Adapter;
 
-use Cake\Network\Http\Adapter\Stream;
-use Cake\Network\Http\Request;
+use Cake\Http\Client\Adapter\Stream;
+use Cake\Http\Client\Request;
```

Comment:
Can these IDE errors be solved by declaring the namespace above all the `class_alias` files? I tested using:

**src/Network/Http/Request.php**
```php
<?php
namespace Cake\Network\Http;

// @deprecated Backwards compatibility with earler 3.x versions.
class_alias('Cake\Http\Client\Request', 'Cake\Network\Http\Request');
```

In PHPStorm, it won't show the red marks for `Cake\Network\Http\Request` now, and loading that class still loads the `Cake\Http\Client\Request`.